main.py

```diff
@@ -712,6 +712,7 @@ async def get_liveness_interface_pairs(self,
                     "status": lsm.ilsm_b.state,
                     "last_hello_at": lsm.ilsm_b.last_hello_at,
                 },
+                "link_id": entry["interface_a"].link.id,
```

Comment:
@HeriLFIU let's make it a bit safer:


```suggestion
                "link_id": entry["interface_a"].link.id if entry["interface_a"].link else None,
```